queue/src/main/java/io/kestra/queue/AbstractQueue.java

```diff
@@ -0,0 +1,32 @@
+package io.kestra.queue;
+
+import com.google.common.base.CaseFormat;
+import jakarta.annotation.Nullable;
+
+public abstract class AbstractQueue<T extends GenericEvent> {
+    protected final Class<T> cls;
+    protected final QueueUtils queueUtils;
```

Comment:
I find it strange to have the QueueUtils here but never us it. It's bad encapsulation should should only defined dependencies you use